docs/iris/src/whatsnew/latest.rst

```diff
@@ -13,193 +13,255 @@ This document explains the changes made to Iris for this release
+  into cubes. This covers most known operational use-cases. (:pull:`3647`)
 
-* Statistical operations :meth:`iris.cube.Cube.collapsed`,
-  :meth:`iris.cube.Cube.aggregated_by` and :meth:`iris.cube.Cube.rolling_window`
-  previously removed every :class:`iris.coord.CellMeasure` attached to the
-  cube.  Now, a :class:`iris.coord.CellMeasure` will only be removed if it is
-  associated with an axis over which the statistic is being run.
+* `stephenworsley`_ improved the handling of :class:`iris.coord.CellMeasure` in
+  the statistical operations :meth:`iris.cube.Cube.collapsed`,
+  :meth:`iris.cube.Cube.aggregated_by` and
+  :meth:`iris.cube.Cube.rolling_window`. These previously removed every
+  :class:`iris.coord.CellMeasure` attached to the cube.  Now, a
+  :class:`iris.coord.CellMeasure` will only be removed if it is associated with
+  an axis over which the statistic is being run. (:pull:`3549`)
 
-* Supporting ``Iris`` for both ``Python2`` and ``Python3`` resulted in pinning our
-  dependency on `Matplotlib`_ at ``v2.x``.  Now that ``Python2`` support has
-  been dropped, ``Iris`` is free to use the latest version of `Matplotlib`_.
+* `stephenworsley`_, `Patrick Peglar`_ and `abooton`_ added support for
+  `CF Ancillary Data`_ variables, which can be loaded from and saved to
+  NetCDF-CF files. Support for `Quality Flags`_ is also provided to ensure they
+  load and save with appropriate units. (:pull:`3800`)
 
-* `CF Ancillary Data`_ variables are now supported, and can be loaded from and
-  saved to NetCDF-CF files. Support for `Quality Flags`_ is also provided to
-  ensure they load and save with appropriate units. See :pull:`3800`.
+* `Bouwe Andela`_ implemented lazy regridding for the
+  :class:`~iris.analysis.Linear`, :class:`~iris.analysis.Nearest`, and
+  :class:`~iris.analysis.AreaWeighted` regridding schemes. (:pull:`3701`)
 
-* Lazy regridding with the :class:`~iris.analysis.Linear`,
-  :class:`~iris.analysis.Nearest`, and
-  :class:`~iris.analysis.AreaWeighted` regridding schemes.
-  See :pull:`3701`.
-
-
-Dependency Updates
-==================
-
-* Iris now supports the latest version of `Proj <https://github.com/OSGeo/PROJ>`_.
-
-* Iris now requires `Cartopy <https://github.com/SciTools/cartopy>`_ >= 0.18 in
-  order to remain compatible with the latest version of `Matplotlib`_.
-
-* GDAL is removed from the extensions dependency group. We no longer consider it to
-  be an extension.
-
-* Configuring Iris and :ref:`installing_from_source` as a developer, with all the
-  required package dependencies is now easier with our curated conda environment
-  YAML files. See :pull:`3812`.
 
 Bugs Fixed
 ==========
 
-* The method :meth:`~iris.Cube.cube.remove_coord` would fail to remove derived
-  coordinates, will now remove derived coordinates by removing aux_factories.
+* `stephenworsley`_ fixed :meth:`~iris.Cube.cube.remove_coord` to now also
+  remove derived coordinates, by removing aux_factories. (:pull:`3641`)
```

Comment:
The comma here should be removed.

Reply:
d1038c403d23de9e461e3fe5e3a94cb7def73778